[TASK] Add Setting Up BE Groups section for Permissions Managemend docs (#330)
TYPO3-Documentation/TYPO3CMS-Tutorial-GettingStarted: Documentation/PermissionsManagement/SettingUpBackendGroups/Index.rst

.. _access-control-lists:

Access Control Lists
--------------------

**ACCESS_CONTROL** or **ACL_**
    Examples: *ACL_content_elements*, *ACL_news*, *ACL_news_extended*, *ACL_module_reports*

    These groups will constitute the largest segment among all, defining granular
    access to content elements, plugins, modules, fields and more.

.. _file-operations:

File Operations
---------------

**FILE_OPERATION_** or **FO_**
    Examples: *FO_all*, *FO_read_write*

    Defines the range of allowed operations for files and folders, such as read, write, delete, etc.

.. _limit-to-languages:

Limit to languages
------------------

**LANGUAGE_**
    Examples: *L_all*, *L_english_german*, *L_en_pl_de*

    Specifies the list of languages available for managing content. Keep in mind
    that you would have to have access to the source language when creating the translation.

As you can see we now have dedicated groups prefixes for Pages access,
Database Mounts, File Mounts, File Operations, Category Mounts and access
to modules, tables, widgets, languages etc. Remember, these are just examples,
and you could — and should — tailor them to meet your specific needs.

> Each group name should be simple and self explaining (tell what permissions it contains).

Reviewer: ```suggestion
Each group name should be simple and sel-explanatory (describing which permissions it contains).
```

.. figure:: ../../Images/ManualScreenshots/PermissionsManagement/PermissionsManagementPrefixedGroupName.png
   :alt: Prefixed group names

   Prefixing group names makes them more organized and easier to search within forms

.. note::
    Use prefixes or other naming conventions which will allow you to easily
    distinguish backend user groups by their purpose.

> Here’s a useful tip for those managing backend groups in your project.

Reviewer 2: ```suggestion
Here’s a useful tip for managing backend groups in your project.
```

> If you’re implementing naming conventions for groups, you likely have documentation

Reviewer 2: ```suggestion
If have decided to implement naming conventions for groups, you probably have detailed
```

> detailing these guidelines. Instead of directing your colleagues to consult

Reviewer 2: ```suggestion
this in your documentation. Instead of directing your colleagues to
```

the documentation for this information, consider extending the TCA for

> the be_groups and be_users tables (and potentially others where the group

Reviewer 2: ```suggestion
the be_groups and be_users tables (and other tables where the group
```

selection box appears). You can add a field description that outlines
the naming conventions and prefixes used. This approach ensures that anyone

> modifying backend group inheritance or assignments will have immediate access

Reviewer 2: ```suggestion
modifying backend group inheritance or assignments will immediately see
```

> to explanations of the naming conventions directly within the interface.

Reviewer 2: ```suggestion
the explanation of the naming convention.
```

.. code-block:: php
   :caption: Add description to a form field through TCA

>    $GLOBALS['TCA']['be_users']['columns']['usergroup']['description'] = 'Prefixes: R_ - Role, PG_ - Page Group, DBM_ - Database Mount, FM_ - File Mount, FO_ - File Operations, CM_ - Category Mount, ACL_ - Access Control';

Reviewer: Long code blocks lead to horizontal scrolling, distribute on several lines
